index.js

```diff
@@ -345,6 +345,13 @@
     function(typeRep) { return []; }
   );
 
+  //  Ordering :: Type
+  var Ordering = $.EnumType(
+    'sanctuary/Ordering',
+    readmeUrl('Ordering'),
+    ['LT', 'EQ', 'GT']
+  );
```

Comment:
Should `Ordering` be exposed to the user?

Pros:
1. Ability to change the values underlying the enumeration in the future without breaking.
2. Documentation of `Ordering`.